stdlib/3/typing.pyi

```diff
@@ -49,6 +49,12 @@ if sys.version_info >= (3, 8):
 if sys.version_info < (3, 7):
     class GenericMeta(type): ...
 
+if sys.version_info >= (3, 10):
+    class ParamSpec:
+        __name__: str
+        def __init__(self, name: str) -> None: ...
+    Concatenate: _SpecialForm = ...
```

Comment:
The `= ...` is unnecessary (but harmless).

Reply:
It's included everywhere else in this file, so we should be consistent.